src/test/resources/mappings/circulation.json

```diff
@@ -11,6 +11,19 @@
           "Content-Type": "application/json"
         }
       }
+    },
+    {
+      "request": {
+        "method": "GET",
+        "url": "/circulation/requests/queue/0b96a642-5e7f-452d-9cae-9cee66c9a892"
```

Comment:
x-okapi-token/x-okapi-tenant will be useful here just to be sure that correct values were populated during FeignClient processing.